src/components/PopularItemCarousel/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
+import CarouselItem from './Item'
+
+const PopularItemCarousel = () => {
```

Comment:
How about naming it this way since we are making it reusable.

```suggestion
const Carousel = () => {
```